backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubUploaderService.java

```diff
@@ -22,52 +28,125 @@
 import javax.persistence.NoResultException;
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
+import java.sql.Timestamp;
 import java.text.SimpleDateFormat;
+import java.time.Instant;
 import java.time.format.DateTimeParseException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
+import java.time.temporal.ChronoUnit;
+import java.util.*;
```

Comment:
I would like to avoid * imports if possible. I find it harder to read the code with them and it clutters the namespace in a way that is harder to debug i.e. importing List under a different path and not seeing the namespace conflict

Reply:
My IDE is doing this automatically when there are more than _N_ includes. I'll change the setting.